<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta
      name="description"
      content="Contig-Plotting Visualization for the Analysis of Short-Read Nucleotide Sequence Alignments"
    />
    <meta name="author" content="2020112553 김민섭(biyotteu)" />
    <meta name="designer" content="유정은" />

    <title>cPlot</title>
    <!-- Pretendard CDN -->
    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
    />
    <!-- Main CSS -->
    <link
      rel="stylesheet"
      href="/~gangman/assets/css/main.css"
    /><!--로컬에서 테스트 시에는 /~gangman빼기-->
  </head>
  <body>
    <header
  class="default-header header"
>
  <nav class="navbar container">
    <a href="/~gangman/" class="logo-link">
      <img
        src="/~gangman/assets/images/logos/logo-black.png"
        class="logo"
        alt="logo"
      />
    </a>
    <button class="hamburger-menu" aria-label="메뉴 열기">
      <span></span>
      <span></span>
      <span></span>
    </button>
    <ul class="nav-links">
      <div class="menu-logo-container">
        <a href="/~gangman/" class="logo-link">
          <img
            src="/~gangman/assets/images/logos/logo-black.png"
            class="logo"
            alt="logo"
          />
        </a>
        <button class="hamburger-menu" aria-label="메뉴 열기">
          <span></span>
          <span></span>
          <span></span>
        </button>
      </div>
      <li class="item"><a href="/~gangman/research">Research</a></li>
      <li class="item"><a href="/~gangman/professor">Professor</a></li>
      <li class="item"><a href="/~gangman/students">Students</a></li>
      <li class="item"><a href="/~gangman/publication">Publication</a></li>
      <li class="item"><a href="/~gangman/courses">Courses</a></li>
      <li class="item"><a href="/~gangman/gallery">Gallery</a></li>
      <!-- <li class="item contact-item">
        <a href="mailto:gangman@dongguk.edu" class="contact-link">
          <img
            src="/~gangman/assets/images/icons/mail-black.png"
            class="contact"
            alt="contact"
          />
        </a>
      </li> -->
    </ul>
    <!-- <a href="mailto:gangman@dongguk.edu" class="contact-link">
      <img
        src="/~gangman/assets/images/icons/mail-black.png"
        class="contact"
        alt="contact"
      />
    </a> -->
  </nav>
</header>

    <!-- 헤더 포함 -->
    <main>
      <div class="research-detail container">
  <article class="research-content">
    <div class="research-entry">
      <header class="research-header">
        <h2>cPlot</h2>
        <div class="research-meta">
          <time datetime="2022-09-21"
            >2022.09.21</time
          >
          
          <span class="author">Gangman Yi</span>
          
        </div>
      </header>

      <div class="research-body"><p><img src="/~gangman/assets/images/research/cplot/image.png" alt="cPlot" /></p>

<h3 id="cplot-contig-plotting-visualization-for-the-analysis-of-short-read-nucleotide-sequence-alignments">cPlot: Contig-Plotting Visualization for the Analysis of Short-Read Nucleotide Sequence Alignments</h3>

<p>Advances in the next-generation sequencing technology have led to a dramatic decrease in read-generation cost and an increase in read output. Reconstruction of short DNA sequence reads generated by next-generation sequencing requires a read alignment method that reconstructs a reference genome. In addition, it is essential to analyze the results of read alignments for a biologically meaningful inference. However, read alignment from vast amounts of genomic data from various organisms is challenging in that it involves repeated automatic and manual analysis steps. We, here, devised cPlot software for read alignment of nucleotide sequences, with automated read alignment and position analysis, which allows visual assessment of the analysis results by the user. cPlot compares sequence similarity of reads by performing multiple read alignments, with FASTA format files as the input. This application provides a web-based interface for the user for facile implementation, without the need for a dedicated computing environment. cPlot identifies the location and order of the sequencing reads by comparing the sequence to a genetically close reference sequence in a way that is effective for visualizing the assembly of short reads generated by NGS and rapid gene map construction.</p>

<p><a href="https://bigdata.dongguk.edu/cPlot" target="\_blank">https://bigdata.dongguk.edu/cPlot</a></p>

<p><img src="/~gangman/assets/images/research/cplot/image1.png" alt="cPlot Introduction" /></p>
</div>

      <div class="research-navigation">
        <div class="nav-links">
          
          <a href="/research/deeppi/" class="prev">‹ Prev</a>
          

          <span class="separator">|</span>

          
          <a href="/research/gene-annotation/" class="next">Next ›</a>
          
        </div>

        <a href="/~gangman/research" class="back">Back to list</a>
      </div>
    </div>
  </article>
</div>

      <!-- 페이지별 내용 -->
    </main>
    <footer>
  <div class="container footer-container">
    <div class="footer-wrap">
      <img
        src="/~gangman/assets/images/logos/gray_dgu_logo.png"
        alt="동국대학교 로고"
      />
      <p>
        신공학관 연구실 #9114, 동국대학교 [04620] 서울특별시 중구 필동로 1길 30
      </p>
    </div>
    <div class="footer-wrap">
      <p>Copyright <strong>&copy;2024 DataLab.</strong> All Rights Reserved</p>
    </div>
  </div>
</footer>

    <!-- 푸터 포함 -->
    <!-- <script src="/~gangman/assets/js/main.js"></script> -->
    <script src="/~gangman/assets/js/navigation.js"></script>
    <script src="/~gangman/assets/js/pagination.js"></script>
    <!--로컬에서 테스트 시에는 /~gangman빼기-->
  </body>
</html>
