<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta
      name="description"
      content="A Robust Method for Finding the Automated Best Matched Genes Based on Grouping Similar Fragments of Large-Scale References for Genome Assembly"
    />
    <meta name="author" content="2020112553 김민섭(biyotteu)" />
    <meta name="designer" content="유정은" />

    <title>Genome Search Plotter</title>
    <!-- Pretendard CDN -->
    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
    />
    <!-- Main CSS -->
    <link
      rel="stylesheet"
      href="/~gangman/assets/css/main.css"
    /><!--로컬에서 테스트 시에는 /~gangman빼기-->
  </head>
  <body>
    <header
  class="default-header header"
>
  <nav class="navbar container">
    <a href="/~gangman/" class="logo-link">
      <img
        src="/~gangman/assets/images/logos/logo-black.png"
        class="logo"
        alt="logo"
      />
    </a>
    <button class="hamburger-menu" aria-label="메뉴 열기">
      <span></span>
      <span></span>
      <span></span>
    </button>
    <ul class="nav-links">
      <div class="menu-logo-container">
        <a href="/~gangman/" class="logo-link">
          <img
            src="/~gangman/assets/images/logos/logo-black.png"
            class="logo"
            alt="logo"
          />
        </a>
        <button class="hamburger-menu" aria-label="메뉴 열기">
          <span></span>
          <span></span>
          <span></span>
        </button>
      </div>
      <li class="item"><a href="/~gangman/research">Research</a></li>
      <li class="item"><a href="/~gangman/professor">Professor</a></li>
      <li class="item"><a href="/~gangman/students">Students</a></li>
      <li class="item"><a href="/~gangman/publication">Publication</a></li>
      <li class="item"><a href="/~gangman/courses">Courses</a></li>
      <li class="item"><a href="/~gangman/gallery">Gallery</a></li>
      <!-- <li class="item contact-item">
        <a href="mailto:gangman@dongguk.edu" class="contact-link">
          <img
            src="/~gangman/assets/images/icons/mail-black.png"
            class="contact"
            alt="contact"
          />
        </a>
      </li> -->
    </ul>
    <!-- <a href="mailto:gangman@dongguk.edu" class="contact-link">
      <img
        src="/~gangman/assets/images/icons/mail-black.png"
        class="contact"
        alt="contact"
      />
    </a> -->
  </nav>
</header>

    <!-- 헤더 포함 -->
    <main>
      <div class="research-detail container">
  <article class="research-content">
    <div class="research-entry">
      <header class="research-header">
        <h2>Genome Search Plotter</h2>
        <div class="research-meta">
          <time datetime="2017-09-13"
            >2017.09.13</time
          >
          
          <span class="author">Gangman Yi</span>
          
        </div>
      </header>

      <div class="research-body"><p><img src="/~gangman/assets/images/research/genome-search-plotter/image.png" alt="Genome Search Plotter" /></p>

<h3 id="a-robust-method-for-finding-the-automated-best-matched-genes-based-on-grouping-similar-fragments-of-large-scale-references-for-genome-assembly">A Robust Method for Finding the Automated Best Matched Genes Based on Grouping Similar Fragments of Large-Scale References for Genome Assembly</h3>

<p>Big data research on genomic sequence analysis has accelerated considerably with the development of next-generation sequencing. Currently, research on genomic sequencing has been conducted using various methods, ranging from the assembly of reads consisting of fragments to the annotation of genetic information using a database that contains known genome information. According to the development, most tools to analyze the new organelles’ genetic information requires different input formats such as FASTA, GeneBank (GB) and tab separated files. The various data formats should be modified to satisfy the requirements of the gene annotation system after genome assembly. In addition, the currently available tools for the analysis of organelles are usually developed only for specific organisms, thus the need for gene prediction tools, which are useful for any organism, has been increased. The proposed method—termed the genome_search_plotter—is designed for the easy analysis of genome information from the related references without any file format modification. Anyone who is interested in intracellular organelles such as the nucleus, chloroplast, and mitochondria can analyze the genetic information using the assembled contig of an unknown genome and a reference model without any modification of the data from the assembled contig.</p>

<p><a href="https://bigdata.dongguk.edu/gene_project/genome_search_plotter" target="\_blank">https://bigdata.dongguk.edu/gene_project/genome_search_plotter</a></p>

<p><img src="/~gangman/assets/images/research/genome-search-plotter/image1.png" alt="Genome Search Plotter" /></p>

<h3 id="input">Input</h3>

<p><strong>Reference accesion Number:</strong></p>

<ul>
  <li>In order to run the BLAST with query contigs and reference, Reference sequences are required. For more information, please refer to NCBI site</li>
</ul>

<p><strong>Input File Format:</strong></p>

<ul>
  <li>Input file is the assembled contigs and format should be in FASTA format. The number of contigs are not limimted, but, it will takes several hours. We recommend to copy the reslut page URL for your reference.</li>
</ul>

<p><strong>Maximum number of matched BLAST hit group:</strong></p>

<ul>
  <li>The number sets the maximum groups which is based on the BLAST result.</li>
</ul>

<p><strong>Minimum number of matched sub gene’s count per each contig:</strong></p>

<ul>
  <li>Please refer to below Figure 1-B.</li>
</ul>

<p><img src="/~gangman/assets/images/research/genome-search-plotter/image2.png" alt="Input Example" /></p>

<p><em>Figure 1. Maximum number of hit group and mininum number of subgene’s count</em></p>

<h3 id="output">Output</h3>

<p>The provided files are “sorted sequences” and “PDF” files</p>

<p><strong>Sorted by the number of subgene:</strong></p>

<ul>
  <li>BLAST result file which is sorted by BLAST e-value. In the sorted query sequences file, sequences are sorted by the number of matched sub-genes and the sequences that do not meet the minimum value of k are filtered out</li>
</ul>

<p><strong>PDF file:</strong></p>

<ul>
  <li>The results are shown as a graph depicting matches with the 149 reference genomes on the X-axis and query sequences uploaded by the user on the Y-axis. Each 150 line on the plot indicates that a query contig is matched with the reference sequences.</li>
</ul>
</div>

      <div class="research-navigation">
        <div class="nav-links">
          
          <a href="/~gangman/research/geneco/" class="prev">‹ Prev</a>
          

          <span class="separator">|</span>

          
          <a href="/~gangman/research/sclassify/" class="next">Next ›</a>
          
        </div>

        <a href="/~gangman/research" class="back">Back to list</a>
      </div>
    </div>
  </article>
</div>

      <!-- 페이지별 내용 -->
    </main>
    <footer>
  <div class="container footer-container">
    <div class="footer-wrap">
      <img
        src="/~gangman/assets/images/logos/gray_dgu_logo.png"
        alt="동국대학교 로고"
      />
      <p>
        신공학관 연구실 #9114, 동국대학교 [04620] 서울특별시 중구 필동로 1길 30
      </p>
    </div>
    <div class="footer-wrap">
      <p>Copyright <strong>&copy;2024 DataLab.</strong> All Rights Reserved</p>
    </div>
  </div>
</footer>

    <!-- 푸터 포함 -->
    <!-- <script src="/~gangman/assets/js/main.js"></script> -->
    <script src="/~gangman/assets/js/navigation.js"></script>
    <script src="/~gangman/assets/js/pagination.js"></script>
    <!--로컬에서 테스트 시에는 /~gangman빼기-->
  </body>
</html>
